external/deployments/rskMainnet/ZeroPriceFeedRevertOnStalePrice.json

```
@@ -0,0 +1,198 @@
+{
+    "address": "0x391fe8a92a7FC626A25F30E8c19B92bf8BE37FD3",
```

this is the PriceFeedMoC address.  
should be replaced with the Zero PriceFeed_Proxy address: 

```suggestion
    "address": "0x6D1d9574d67e04cf35Fa1d916F763eDDae03b75d",
```

---

Not relevant anymore with the latest changes